### ISSUE TITLE
Unknown span and character in editor

### ISSUE THREAD

**Issue by joebrumley** [getgrav/grav-plugin-admin#1233]
![screenshot 2017-09-27 12 30 25](https://user-images.githubusercontent.com/296942/30925386-b03b5f6c-a37f-11e7-9fa0-eb2ff9500ebd.png)

I have been experiencing an issue with the editor adding an extra character and a non-breaking space in a span. I have checked the pages outside of the editor and they do not contain the character.

![screenshot 2017-09-27 12 28 07](https://user-images.githubusercontent.com/296942/30925286-5b0fa944-a37f-11e7-9c99-0c4dc343c48c.png)

I'm not sure if this is an issue with Admin plugin or not but I thought I would start here. It has been happening for quite some time and each time I upgrade Grav and/or the Admin plugin it still persists. If anyone has an idea of what may be happening or a setting that I am missing, I would greatly appreciate the guidance.

Thanks.

**Comment by rhukster** [comment]
You have a special character in your content somehow.  Did you create this in the editor or some other tool?

**Comment by joebrumley** [comment]
Thanks for the quick reply rhukster. I did not create the content in another editor. It happens when I create new content in the Admin interface. It does also appear if I create the content in vi or another editor as well.

**Comment by rhukster** [comment]
What browser is this? I cannot replicate if i create content in Grav Admin or an offline editor.

**Comment by joebrumley** [comment]
I am using Chrome but it also happens in Safari. Here is a list of the plugins that I have installed. I will try with a fresh install to see if that fixes the issue.

![screenshot 2017-09-27 13 24 38](https://user-images.githubusercontent.com/296942/30927748-7a34bd16-a387-11e7-9e28-900301566dc0.png)

**Comment by rhukster** [comment]
You can also try disabling plugins one a time to see if you can determine which one it is.  Frankly though, I have most of those installed on my test bed setup (except `instagram` and `randomizer`)

**Comment by joebrumley** [comment]
Thanks for checking this out with me.

Here is what I get after a fresh install with no additional plugins beyond what Admin uses:

![screenshot 2017-09-27 14 04 59](https://user-images.githubusercontent.com/296942/30929604-f53f1bd2-a38c-11e7-97a1-0b4d10da2ebc.png)

**Comment by rhukster** [comment]
Ok did you 'save' anything or is that just how it looks when you open the file in the admin plugin editor for the firs time?

**Comment by joebrumley** [comment]
That's what it looked like without saving anything. I opened the page in the editor and there it was.

**Comment by w00fz** [comment]
Double check that your browser and php are set to UTF-8

**Comment by joebrumley** [comment]
Ok. In Safari I was able to specifically set the browser to UTF-8 from Default and it got rid of the character. Thank you!! Now I have to figure out what Default is and how to make it get passed from PHP. Thanks guys for your help!

**Comment by joebrumley** [comment]
It looks like it is fixed now. Switching the browser text-encoding was fine until I closed and reopened the browser. What I did that works after closing the browwer is the following - 

**Apache configuration (in httpd.conf or .htaccess)**
`AddDefaultCharset` utf-8

- Which I added to the Virtual Host configuration for my  site

**PHP (in php.ini)**
```
default_charset = "utf-8"
mbstring.internal_encoding=utf-8
mbstring.http_output=UTF-8
mbstring.encoding_translation=On
mbstring.func_overload=6 
```
I created a custom.ini file for this and enabled it with phpenmod. 

Now when I refresh the site it all works. I'm not sure which one did the trick but I figure it can't hurt to have both.

Thanks for pointing me down the utf-8 path w00fz. I will close this issue.
